optaplanner-docs/src/main/asciidoc/Chapter-Quick_start/Chapter-Quick_start.adoc

```diff
@@ -71,160 +69,27 @@ image::Chapter-Quick_start/cloudOptimizationValueProposition.png[align="center"]
 [[cloudBalancingDomainModelDesign]]
 === Domain Model Design
 
-Beginning with the domain model:
 
+Using a <<domainModelingGuide,domain model>> helps determine which classes are planning entities and which of their properties are planning variables. Using a domain model helps to simplify contraints, improve performance, and increase flexibility for future needs. 
+
+//procedure from chapter 17 - Design Patterns
+
+. Draw a class diagram of your domain model. 
+. Normalize it to remove duplicate data.
+. Write down some sample instances for each class.
```

Comment:
I hope the change I have made here is right and makes sense. I just went from the image we are referring to for this section.